src/Admin/AbstractAdmin.php

```diff
@@ -1871,6 +1877,20 @@ public function getChild($code)
     public function setParent(AdminInterface $parent)
```

Comment:
Can you please add PHPDoc for the new second parameter

Reply:
I've been looking for a way to do that, but I haven't found any, any idea? I tried with `@method`, but it looked like the IDE didn't recognize it.